src/Common/tests/System/Diagnostics/RemoteExecuteForUap.cs

```diff
@@ -0,0 +1,86 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why not instead add this to CoreFx.Private.TestUtilities so that it's available by default without a Link?

Reply:
Same for the ThreadCultureChange.

Reply:
> Why not instead add this to CoreFx.Private.TestUtilities so that it's available by default without a Link?

We have a lot of code in our common test folder.  What's the rationale for what goes in there to be included file-by-file vs being included in CoreFx.Private.TestUtilities?

Reply:
I changed it per your request, but I'd like to understand the rationale for your suggestion.  Are you planning to move all the other helpers in Common\tests into CoreFx.Private.TestUtilities?

Reply:
The helper files and the strategy to link them into the test projects directly existed long before CoreFx.Private.TestUtilities was added (during the UWP bringup phase in 2017). I think every helper which exists in more than one test project should live in the TestUtilities library. Opened https://github.com/dotnet/corefx/issues/41785

Reply:
Thanks